src/quo/components/wallet/progress_bar/view.cljs

```diff
@@ -2,7 +2,19 @@
   (:require
     [quo.components.wallet.progress-bar.style :as style]
     [quo.theme :as quo.theme]
-    [react-native.core :as rn]))
+    [react-native.core :as rn]
+    [schema.core :as schema]))
+
+(def ?schema
+  [:=>
+   [:catn
+    [:props
+     [:map {:closed false}
```

Comment:
Map schemas are open by default, thus we can remove the explicit `:closed` option.